data/tutorials/getting-started/1_00_install_OCaml.md

```diff
@@ -88,42 +88,53 @@ Opam is now installed and configured! You can now move to [installing the OCaml
 
 ## Installation on Windows
 
-In this section, we'll describe using the [DkML](https://gitlab.com/dkml/distributions/dkml#installing) Windows distribution. Expect to see another officially-supported Windows installation provided directly by opam in the coming months; it will be compatible with your DkML installation.
```

Comment:
A major part of the effort with opam 2.2.0 has been to make it that the experience installing on Windows is now equivalent: i.e. it's `package-manager install opam`. It would seem better to me to merge these instructions, which I've proposed in https://github.com/dra27/ocaml.org/commit/16ba45f75b212bb4d3018a72db39315045cd542b (feel free to cherry-pick if you agree)